keyboards/3.0/bn.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard3 SYSTEM "../dtd/ldmlKeyboard3.dtd">
+<keyboard3 locale="bn" conformsTo="techpreview">
+    <!--
+        History:
+          Based on
+    https://github.com/keymanapp/ldml-keyboards-dev/blob/master/keyboards/sil-bengali/bn-t-k0-cldr-phonetic.ldml
+     -->
+    <locales>
+        <locale id="as" />
+        <locale id="syl" />
+    </locales>
+    <version number="1.3.0" />
+    <info name="SIL Bengali-Assamese Phonetic" indicator="bn" layout="QWERTY" />
+
+    <keys>
+        <import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml" />
+        <import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+
+        <!-- key names based on https://www.loc.gov/catdir/cpso/romanization/bengali.pdf + Assamese for /wa/ -->
+
+        <!-- UNSHIFTED KEYS -->
+        <!-- E: (top) row -->
+        <key id="n̐" output="\u{0981}" /> <!-- candrabindu-->
+        <key id="1" output="১" />
```

Comment:
So this keyboard cannot output western 0-9 digits?

Reply:
That's what it came with. Could be on an alt key though